src/test/java/uk/nhs/hee/tis/trainee/forms/service/FormRPartAServiceImplTest.java

```diff
@@ -133,7 +145,19 @@ void shouldSaveSubmittedFormRPartA() {
     assertThat("Unexpected trainee ID.", savedDto.getTraineeTisId(), is(DEFAULT_TRAINEE_TIS_ID));
     assertThat("Unexpected forename.", savedDto.getForename(), is(DEFAULT_FORENAME));
     assertThat("Unexpected surname.", savedDto.getSurname(), is(DEFAULT_SURNAME));
-    verify(s3Mock).putObject(any(PutObjectRequest.class));
+    verify(s3Mock).putObject(putRequestCaptor.capture());
```

Comment:
nice

Reply:
not entirely sure but the assertions around the `Unexpected Bucket Name.` bits sound like its part of an unhappy path test but the method name here `shouldSaveSubmittedFormRPartA` sounds like happy path

Reply:
I stuck with the convention `Unexpected {FIELD_NAME}.` for the display message if the assertion fails.  I did have to read it a second time.